### DISCUSSION TITLE
auto reset

### DISCUSSION THREAD

**Discussion by PandaKeny** [SudhanPlayz/Discord-MusicBot#849] [Q&A] [answered]
If the bot works for too long, it leads to a lavalink error, can you add the time for the file to auto reset?

**Comment by brianferri** [comment]
Scripts like that can vary depending on the OS you're running it on, also please check if it isn't a hosting problem instead. Check if the problem originates from the Lavalink or the Bot itself not being able to make the connection.
Attach a log of the error or a screenshot of the console if possible, as well as the version of the bot you're using, please.

**Comment by PandaKeny** [comment]
I'm running windows server 2019 operating system, the bot was normal at first, but it took about a week to use the command, it reported an error that could not connect to lavalink, after resetting the bot, it was normal

**Comment by malobaidan** [comment]
Same problem, host on ubuntu 21.04

**Comment by JotaroKujo0525** [comment]
Well, the lavalink is the problem ¯\_(ツ)_/¯

**Comment by PandaKeny** [comment]
Or is there a command to reset the bot (/reload) ?

**Comment by JotaroKujo0525** [comment]
Well technically if the lavalink is brokey, even if you reload it, it will still be like that

**Comment by JotaroKujo0525** [comment]
also, there's no such thing as "resetting the bot". Yes you can reload the commands, in fact there's a command like that in v5 iirc. If you mean resetting the code itself, well you cant, the code will be killed entirely. But if you meam relogging the bot to discord.js (basically logging out and then login in again) well you can but that won't solve the lavalink issue either, it can only solve the Slash Commands, and some stuff

**Comment by JotaroKujo0525** [comment, marked as answer]
the best thing to do is to  **find or host a good lavalink**
